src/PasswordResetController.js

```diff
@@ -45,6 +45,41 @@ function (Okta, FormController, FormType, ValidationUtil, FooterSignout, TextBox
     Form: {
       save: Okta.loc('password.reset', 'login'),
       title: Okta.loc('password.reset.title', 'login'),
+      subtitle: function () {
+        var policy = this.options.appState.get('policy');
+        if (!policy || !policy.complexity) {
+          return;
+        }
+
+        var fields = {
+          minLength: {i18n: 'password.complexity.length', args: true},
+          minLowerCase: {i18n: 'password.complexity.lowercase'},
+          minUpperCase: {i18n: 'password.complexity.uppercase'},
+          minNumber: {i18n: 'password.complexity.number'},
+          minSymbol: {i18n: 'password.complexity.symbol'},
+          excludeUsername: {i18n: 'password.complexity.no_username'}
+        };
+        var requirements = [];
+        var complexity = policy.complexity;
+
+        _.each(fields, function (params, complexityType) {
+          if (complexity[complexityType]) {
+            params.args ?
+              requirements.push(Okta.loc(params.i18n, 'login', [complexity[complexityType]])) :
+              requirements.push(Okta.loc(params.i18n, 'login'));
+          }
+        });
```

Comment:
We can use a map on the complexity object instead of iterating over fields:
```javascript
var requirements = _.map(complexity, function (val, complexityType) {
  var field = fields[complexityType];
  return field.args ? Okta.loc(field.i18n, 'login', [val]) : Okta.loc(field.i18n, 'login');
}
```